stdlib/public/core/Integers.swift.gyb

```diff
@@ -2646,6 +2646,29 @@ public struct ${Self}
     return Bool(Builtin.cmp_${u}lt_Int${bits}(lhs._value, rhs._value))
   }
 
+  // FIXME(integers): these overloads are here in order for the compiler to
+  // prefer them to the generic ones on BinaryInteger, in cases like the
+  // following, where choosing Int as a literal type leads to overflow.
```

Comment:
I'll wait for all the triggered builds to succeed and then commit the missing code block.